com.unity.render-pipelines.universal/ShaderLibrary/MetaInput.hlsl

```diff
@@ -1,70 +1,85 @@
 #ifndef UNIVERSAL_META_PASS_INCLUDED
 #define UNIVERSAL_META_PASS_INCLUDED
 
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/MetaPass.hlsl"
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
 
-CBUFFER_START(UnityMetaPass)
-// x = use uv1 as raster position
-// y = use uv2 as raster position
-bool4 unity_MetaVertexControl;
-
-// x = return albedo
-// y = return normal
-bool4 unity_MetaFragmentControl;
-CBUFFER_END
+struct Attributes
+{
+    float4 positionOS   : POSITION;
+    float3 normalOS     : NORMAL;
+    float2 uv0          : TEXCOORD0;
+    float2 uv1          : TEXCOORD1;
+    float2 uv2          : TEXCOORD2;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
 
-float unity_OneOverOutputBoost;
-float unity_MaxOutputValue;
-float unity_UseLinearSpace;
+struct MetaVaryings
+{
+    float4 positionCS   : SV_POSITION;
+    float2 uv           : TEXCOORD0;
+#ifdef EDITOR_VISUALIZATION
+    float2 VizUV        : TEXCOORD1;
+    float4 LightCoord   : TEXCOORD2;
+#endif
+};
 
-struct MetaInput
+struct Varyings
 {
-    half3 Albedo;
-    half3 Emission;
-    half3 SpecularColor;
+    float4 positionCS   : SV_POSITION;
+    float2 uv           : TEXCOORD0;
 };
 
-float4 MetaVertexPosition(float4 positionOS, float2 uv1, float2 uv2, float4 uv1ST, float4 uv2ST)
```

Comment:
There might be users using this, you could still define this and redirect to UnityMetaVertexPosition for backward compatibility.

Reply:
Fixed in 5b3637f